testgrid/config.yaml

```
@@ -2843,6 +2843,11 @@ dashboards:
     description: "Uses kubetest to run e2e tests (+Feature:Ingress|NEG) against a cluster created with cluster/kube-up.sh"
     alert_options:
       alert_mail_to_addresses: kubernetes-sig-network-test-failures@googlegroups.com
+  - name: skew-cluster-latest-kubectl-stable1-gce
```

Personally this name is hard to parse, what about some thing like?
```suggestion
  - name: gce-kubectl-skew
```